test/fixtures/testmatrix_sensitive.csv

```diff
@@ -5,4 +5,4 @@ Kutya;;;;Allat;ToAll
 Sas;;;;ToAll;Kigyo
 Kutya;macska@example.com;;;Macska;ToAll
 id1;;Scotland;;Britt;Kigyo
-Macska;;USA;;ToAll;Ireland
+Macska;;USA;;ToAll;Ireland
```

Comment:
#minor Accidental change?

Reply:
I downloaded the file directly from their repo 👍